### PR TITLE
Remove browsermobproxy and HAR capture feature

### PR DESCRIPTION
**Pull request by jzoldak**
@thallada @ajpal @jmbowman @dianakhuang

### CODE REVIEW

File: test_requirements.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ coverage==3.7
 nose==1.3.0
 mock==1.0.1
 pep8==1.7.0
+requests==2.10.0
```
Comment (dianakhuang) [inline test_requirements.txt]: Why is this being added?

**Comment by dianakhuang** [comment]
Other than that one question, 🚢 it!

File: test_requirements.txt (hunk 1 above)
Comment (jzoldak) [inline test_requirements.txt]: Previously it got installed "for free" as a subdependency of browsermob-proxy, whose setup.py includes this:
```
install_requires=['requests>=1.1.0'],
```

**Comment by jmbowman** [comment]
👍

File: test_requirements.txt (hunk 1 above)
Comment (jzoldak) [inline test_requirements.txt]: ... and then @jmbowman discovered that it's not even being used in the test file that imported it so we could get rid of it anyhow. Bonus!

**Comment by benpatterson** [comment]
🎆

**Pull request merged**